pom.xml

```diff
@@ -28,8 +28,8 @@
 
         <spring.version>4.3.7.RELEASE</spring.version>
         <spring-cloud.version>Dalston.RELEASE</spring-cloud.version>
-        <spring-cloud-deployer.version>1.2.1.RELEASE</spring-cloud-deployer.version>
-        <spring-cloud-deployer-local.version>1.3.0.M1</spring-cloud-deployer-local.version>
+        <spring-cloud-deployer.version>1.3.0.BUILD-SNAPSHOT</spring-cloud-deployer.version>
```

Comment:
This should be spring-cloud-deployer version 1.3.0.M1?  also for deployer-local it should be 1.3.0.M1 as well.  Can't have snapshots in a milestone release of skipper and there are no additional commits in those repos which we need post 1.3.0.M1 for cloud-deployer and deployer-local.